tray/src/com/dvt/main/creational/abstract_factory/Part.java

```diff
@@ -0,0 +1,9 @@
+<<<<<<< HEAD
+package com.dvt.creational.abstract_factory;
+=======
+package creational.abstract_factory;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.abstract_factory;
=======
package creational.abstract_factory;
>>>>>>> main
```